t/sanity.t

```diff
@@ -86,7 +86,8 @@ dog: 32 (flags: 0)
             local memcached = require "resty.memcached"
             local memc = memcached:new()
 
-            memc:set_timeout(1000) -- 1 sec
```

Comment:
Need to add another test for `set_timeouts` instead of change `set_timeout` one.